### PR TITLE
node 13.12.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/node.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Node < Formula
   desc "Platform built on V8 to build network applications"
   homepage "https://nodejs.org/"
-  url "https://nodejs.org/dist/v13.11.0/node-v13.11.0.tar.xz"
-  sha256 "e5402183e68806785b3c40c2cb0a6b6aa43bb61aee1cec5efde9c65825ef291f"
+  url "https://nodejs.org/dist/v13.12.0/node-v13.12.0.tar.gz"
```
Comment (Bo98) [inline Formula/node.rb]:
```suggestion
  url "https://nodejs.org/dist/v13.12.0/node-v13.12.0.tar.xz"
```

Can we stick with `xz`?

Comment (chenrui333) [reply Formula/node.rb]: yeah, my bad

**Comment by chenrui333** [comment]
only ungit formula failed.

**Comment by chenrui333** [comment]
Move forward with the merge

**Pull request closed without merging**